docs/source/get_hsup_bc_ic.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Command line arguments
+if [ -z "$1" ]; then
+   echo "Usage: $0 yyyymmddhh "
+   echo "For example: 2018100900, 2019052300, 2019071200, 2019092512, 2019102812, 2020011812, 2020020312, 2020040912, 2020072300" 
+   exit
+fi
+## date (year month day) and time (hour)
+yyyymmddhh=$1 #i.e. "2018100900"
+
+if [ "$#" -ge 2 ]; then
+  nfcst=$2 #i.e. "12"
+else
+  nfcst=90
+fi
+
+## forecast interval, the default interval are 3 hours
```

Comment:
default interval is 3 hours

Reply:
Yes, the default interval is 3 hours. Some old data have 6-h interval.

Reply:
Sorry for the misunderstanding, I was trying to correct the language. It should be:
```
## forecast interval, the default interval is 3 hours
```

Reply:
Thanks!